packages/core/test/actions.test.ts

```diff
@@ -1653,7 +1653,7 @@ describe('sendTo', () => {
       context: () => ({
         child: spawn(childMachine)
       }),
-      entry: sendTo((ctx) => ctx.child as any, { type: 'EVENT' })
+      entry: sendTo((ctx) => ctx.child, { type: 'EVENT' })
```

Comment:
we should add a test case validating that `sendTo((ctx) => ctx.child, { type: 'UNKNOWN' })` reports a type error

Reply:
Done in [b956a84](https://github.com/statelyai/xstate/pull/3238/commits/b956a8405e441bf75594852b68ef5f7e19eb1a24)